src/main/java/org/influxdb/InfluxDBException.java

```diff
@@ -18,4 +18,111 @@ public InfluxDBException(final String message, final Throwable cause) {
   public InfluxDBException(final Throwable cause) {
     super(cause);
   }
+
+  /**
+   * @return true if the operation may succeed if repeated, false otherwise.
+   */
+  public boolean isRetryWorth() {
+    return true;
+  }
+
```

Comment:
Could you please add here a comment pointing to where these messages can be found (e.g. InfluxDB source files, documentation, etc)? This would help the next developer who may need to work on this code.

Reply:
It is now documented in the InfluxDBException class source code.

Reply:
But every developer would stumble over this code because he would ask himself what this method is all about. Is there any future modification expected that this method would return something different than true in the near future ? If no, please remove.

Reply:
There is a couple of subclasses of InfluxDBException for specific errors that return a different value in isRetryWorth() already. For example AuthorizationFailedException, DatabaseNotFoundException. I also added a javadoc comment for explanation.